.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,64 @@
+name: "Feature Request"
+description: "Have a great new idea? We'd love to know, please search through our existing feature requests, and upvote it if its already been submitted."
```

Comment:
Shouldn't we ask to first discuss feature requests on Discord?

Reply:
Why? I don't think that this would be needed. If someone has a feat request - they can just submit id and we will reply. I think that forcing people. to use Discord beforehand will be unnecessary overhead. Do you agree?